scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+# The scala server will run on port 9000 by default
+EXPOSE 9000
+
+# nginx port
+EXPOSE 8080 
+
+# Install mocha for API testing
+RUN npm install -g mocha
+
+########################### TEST-8 #############################
+# Install bats (needed for do-exclusively script)
+RUN git clone https://github.com/sstephenson/bats.git \
```

Hadolint

[warning] Use WORKDIR to switch to a directory

(DL3003)